### PR TITLE
Add android . release

### PR DESCRIPTION
**Pull request by strseb**
Note: we're still in review stage, so nothing is live :)

### CODE REVIEW

File: docs/mozillavpn.json [hunk 1]
```diff
@@ -93,6 +93,11 @@
       "build": null,
       "date": "2022-10-25",
       "platforms": ["ANDROID", "LINUX", "MACOS", "WINDOWS", "IOS"]
+    },
+    "2.10.1": {
+      "build": null,
+      "date": "2022-10-26",
+      "platforms": ["ANDROID"]
```
Comment (bakulf) [inline docs/mozillavpn.json:100]: and linux

File: docs/mozillavpn.json [hunk 2]
```diff
@@ -93,6 +93,11 @@
       "build": null,
       "date": "2022-10-25",
       "platforms": ["ANDROID", "LINUX", "MACOS", "WINDOWS", "IOS"]
```
Comment (bakulf) [inline docs/mozillavpn.json:95]: remove linux from here

File: docs/mozillavpn_stage.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "latest": {
-    "ANDROID": "2.10.0",
+    "ANDROID": "2.10.1",
     "IOS": "2.10.0",
     "LINUX": "2.10.0",
```
Comment (bakulf) [inline docs/mozillavpn_stage.json:5]: 2.10.1 for linux

File: docs/mozillavpn_stage.json [hunk 4]
```diff
@@ -68,6 +68,11 @@
       "build": null,
       "date": "2022-10-12",
       "platforms": ["ANDROID", "IOS", "LINUX", "MACOS", "WINDOWS"]
+    },
+    "2.10.1": {
+      "build": null,
+      "date": "2022-10-26",
+      "platforms": ["ANDROID"]
```
Comment (bakulf) [inline docs/mozillavpn_stage.json:75]: add linux

**Review by bakulf — CHANGES_REQUESTED** [review]

**Review by birdsarah — APPROVED** [review]

**Pull request merged**